src/components/pages/Survey/Survey.tsx

```
@@ -23,30 +29,50 @@
-      interestEtc: {},
-      joinReason: "",
-      feedBack: "",
+      interestFields: interestFields,
+      interestEtc: interestEtc,
+      joinReason: joinReason,
+      feedBack: feedBack,
     },
   });
 
-  console.log(methods.getValues());
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: methods.getValues

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)